test/export/test_export.py

```diff
@@ -3249,6 +3249,82 @@ def dynamify_inp(x):
         with self.assertRaisesRegex(RuntimeError, "shape\[0\] to be >= 3, but got 2"):
             ep.module()(*test_inp)
 
+    def test_nested_module(self):
+        class M1(torch.nn.Module):
+            def forward(self, x):
+                return x + x
+
+        class M2(torch.nn.Module):
+            def forward(self, x):
+                m = M1()
+                return m(x) * x
+
+        inps = (torch.randn(3, 3),)
+        ep = export(M2(), inps)
```

Comment:
How about we also assertExpectedInline of the graph? It's ok if strict-mode and non-strict mode has different graph, we can branch on it so that we can clearly see the difference. It's not immediate to see what the graphs look like and what's the difference between strict and nonstrict mode.